_layouts/feedback.html

```diff
@@ -201,7 +199,7 @@ <h2>Overall</h2>
                 <div class="col-md-6">
                     <div class="card">
                         <div class="card-body">
-                            <h5 class="card-title">{{ fdbk | size }} feedback</h5>
+                            <h5 class="card-title">Feedback over time</h5>
```

Comment:
I liked have the number displayed. Could we add it again in the title? Then at least the screen reader can have the number (if not the graphs)

Reply:
sure! I was finding it a bit confusing with the multiple lines since there was an "overall" line and several individual ones, maybe it just needs the right wording or so.